cpp/ycm/ClangCompleter/Diagnostic.h

```diff
@@ -33,6 +33,34 @@ enum DiagnosticKind {
   WARNING
 };
 
+/// Information about a replacement that can be made to the source to "fix" a
+/// diagnostic
+struct FixItChunk {
+  /// the replacement string. this string should replace the source range
```

Comment:
No problem. I'm just so used to things correcting my lazy typing. You'd think there was a vim setting for that... *google*